browser/chrome/browser/browser.dtd

```diff
@@ -743,7 +743,7 @@ you can use these alternative items. Otherwise, their values should be empty. --
 <!ENTITY trackingContentBlocked.block.accesskey "W">
 <!ENTITY trackingContentBlocked.disabled.message "Ochrona przed śledzeniem jest wyłączona">
 
-<!ENTITY pointerLock.notification.message "Naciśnij ESC w dowolnej chwili, aby wyświetlić go ponownie.">
+<!ENTITY pointerLock.notification.message "Naciśnij Esc w dowolnej chwili, aby wyświetlić go ponownie.">
```

Comment:
In [en-US](https://github.com/mozilla/gecko-dev/blame/master/browser/locales/en-US/chrome/browser/browser.dtd#L834) we have "ESC" here but I didn't found a reason, so "Esc" should be fine.